core/src/test/java/io/questdb/griffin/engine/functions/catalogue/PrefixedClassCatalogueFunctionFactoryTest.java

```diff
@@ -182,7 +182,7 @@ public void testPSQLTableList() throws Exception {
                         "  pg_catalog.pg_get_userbyid(c.relowner) as \"Owner\"\n" +
                         "FROM pg_catalog.pg_class c\n" +
                         "     LEFT JOIN pg_catalog.pg_namespace n ON n.oid = c.relnamespace\n" +
-                        "WHERE c.relkind IN ('r','p','v','m','S','f','')\n" +
+                        "WHERE c.relkind IN ('r','p','v','m','S','f',cast('' as char))\n" +
```

Comment:
We must not change the SQL here to match our implementation. This SQL is run by `psql` and has to execute verbatim

Reply:
this breaks `psql` basically

```
io.questdb.griffin.SqlException: [521] CHAR constant expected

	at io.questdb/io.questdb.std.ThreadLocal.initialValue(ThreadLocal.java:36)
	at java.base/java.lang.ThreadLocal.setInitialValue(ThreadLocal.java:195)
	at java.base/java.lang.ThreadLocal.get(ThreadLocal.java:172)
```

Reply:
I see. Then we need to do an implicit cast for empty string literal present in `IN` clause. Going to update the PR.

Reply:
Cool, I would also be careful in the future changing SQL text in these test class, there are some other tests that assert that third-party tools work. Perhaps we should move them into a separate test class and annotate to make the situation more obvious

Reply:
> Perhaps we should move them into a separate test class and annotate to make the situation more obvious

That might be a good idea. Created #1552 to track the task.

Reply:
Done in 19f474a13c782d0fa9dd13be9a22624c14941ac5